File: src/gestorAplicacion/clasesPrincipales/TecnoVans.java
package gestorAplicacion.clasesPrincipales;

import java.util.ArrayList;

public class TecnoVans extends Bus{
	int codigo;
	private final int precio=800;//precio por km
	
	//constructor
		public TecnoVans(String modelo, String placa, Conductor conductor, int capacidad, ArrayList<String> sillas) {
			super(modelo,placa,conductor,sillas);
			this.capacidad=capacidad;
			this.codigo=200;
		}
	//get&set capacidad
		public int getCapacidad() {
			return capacidad;
		}
		public void setCapacidad(int capacidad) {
			this.capacidad=capacidad;
		}
		public int getPrecio() {
			return precio;
		}
		//get&set codigo
		public int getCodigo() { //desde aqui agregue 
			return codigo;
		}

		public void setCodigo(int codigo) {
			this.codigo = codigo;
		}
		@Override
		public void sillasDisponibles() {
			for(Tiquete tiquete : Viaje.gettiquetesTodos()){
				if(tiquete.getEstado() == false){
					System.out.println("la silla: "+tiquete.getSillaTiquete()+"esta disponible");
					}
				}
			}
		public String sillaTiquete(Tiquete tiquete) {
			return tiquete.getSillaTiquete();
		}
}

File: Python/gestor_aplicacion/clasesHerencia/TecnoVans.py
from gestor_aplicacion.clasesPrincipales.Viaje import Viaje
from gestor_aplicacion.clasesPrincipales.Bus import Bus


class TecnoVans(Bus):
    _tecnosv = []

    def __init__(self, placa, conductor,  sillas):
        # instance fields found by Java to Python Converter:
        self._precioKm = 800
        self._capacidad = 20
        self.name = "TecnoVans"
        self.placa = placa
        self.conductor = conductor
        super().__init__(placa, conductor, sillas)
        self.codigo = 200
        TecnoVans._tecnosv.append(self)

    
    def getPlaca(self):
        return self.placa

    def getConductor(self):
        return self.conductor

    # get&set capacidad
    def getCapacidad(self):
        return self._capacidad

    def getPrecioKm(self):
        return self._precioKm
    # get&set codigo

    def getCodigo(self):
        return self.codigo

    def setCodigo(self, codigo):
        self.codigo = codigo

    def sillasDisponibles(self):
        for tiquete in Viaje.getTiquetesTodos():
            if tiquete.getEstado() == False:
                print("la silla: "+tiquete.getSillaTiquete()+"esta disponible")

    def sillaTiquete(self, tiquete):
        return tiquete.getSillaTiquete()
